README.md

```diff
@@ -22,145 +22,178 @@ composer require tobscure/json-api
 
 ```php
 use Tobscure\JsonApi\Document;
-use Tobscure\JsonApi\Collection;
 
-// Create a new collection of posts, and specify relationships to be included.
-$collection = (new Collection($posts, new PostSerializer))
-    ->with(['author', 'comments']);
+$resource = new PostResource($post);
 
-// Create a new JSON-API document with that collection as the data.
-$document = new Document($collection);
+$document = Document::fromData($resource);
```

Comment:
I'm proposing to name method more verbose (because we will pass `Resource` there):
```php
$document = Document::fromResource($resource);
```

Reply:
You can also pass in an array of resources. `fromData` matches with the top-level `data` key in the resulting document.